SPM Example/PortalSwift/ViewController.swift

```diff
@@ -590,26 +733,34 @@ class ViewController: UIViewController, UITextFieldDelegate {
         self.logger.error("Error: Do you have `GDRIVE_CLIENT_ID=$(GDRIVE_CLIENT_ID)` in your Secrets.xcconfig?")
         throw PortalExampleAppError.environmentNotSet()
       }
+      guard let BACKUP_WITH_PORTAL: String = infoDictionary["BACKUP_WITH_PORTAL"] as? String else {
+        self.logger.error("Error: Do you have `BACKUP_WITH_PORTAL=$(BACKUP_WITH_PORTAL)` in your Secrets.xcconfig?")
```

Comment:
```suggestion
        self.logger.error("Error: The environment variable `BACKUP_WITH_PORTAL` is not set or is empty. Please ensure that `BACKUP_WITH_PORTAL=true` or `BACKUP_WITH_PORTAL=false` is included in your Secrets.xcconfig file, and that `BACKUP_WITH_PORTAL=$(BACKUP_WITH_PORTAL)` is referenced correctly in your App's info.plist.")
```
This is more clear and descriptive message.
Same comments are applied to all the other properties we read from the info.plist that comes from the Secrets.xcconfig file.